GopherCAN.c

```diff
@@ -1001,11 +962,11 @@ static void build_message_id(CAN_MSG* msg, CAN_ID* id)
 //  a CAN id struct. No error checking is performed
 static void get_message_id(CAN_ID* id, CAN_MSG* message)
```

Comment:
Might be worth changing the name or adding some comments to note this is the GCAN ID type, not a data ID type